lib/encapsulate_as_money/activemodel_integration.rb

```diff
@@ -0,0 +1,24 @@
+begin
+  require 'active_model'
+
+  if ActiveModel::VERSION::MAJOR == 4
+    module ActiveModel::Validations::Clusivity
+      private
+
+      def inclusion_method(enumerable)
```

Comment:
Rather than poking about the types in the enumerable, is there a way to introspect the enumerable to see if it'll iterate instead?